public/scripts/statusDisplay.js

```diff
@@ -0,0 +1,35 @@
+const displayDiv = document.querySelector(".display-div");
+
+export const updateStatusDisplay = (state) => {
+  let displayStatusMessage = displayDiv.innerText;
+  if (state.gameOverState.isOver) {
+    displayStatusMessage = generateWinnerDisplayMessage(state.gameOverState);
+  } else {
+    displayStatusMessage = generateTurnDisplay(state.players)
+  }
+
+  displayDiv.innerText = displayStatusMessage;
+}
+
+const generateWinnerDisplayMessage = (gameOverState) => {
+  const winner = gameOverState.winner;
+  return (winner === null) ? "It's a tie!" : winner + " wins!";
+}
+
+const generateTurnDisplay = (playersState) => {
+  let currentPlayerMark = playersState.currentPlayerMark;
+  return isHuman(playersState) ? generateHumanTurnMessage(currentPlayerMark) : generateAiTurnmessage(currentPlayerMark); 
+}
+
+const isHuman = (playersState) => {
```

Comment:
Is this a duplicate declaration? Looks like you also have this function in `main.js`. It's also strange that it is within this file as I'd expect the functions in here to be related to `statusDisplay`

Reply:
It was a duplicate declaration, but I was also using the method inside of `statusDisplay.js`. To avoid that, I've imported `#isAi` from `ai.js`